docs/tutorials/getting-started-with-csharp/console-teleprompter/sampleQuotes.txt

```diff
@@ -203,9 +203,6 @@ Funny Quotes
 
 -Woody Allen
 
-Otherwise Intelligent Quotes
```

Comment:
if you are removing this header, should you also remove the one from line 104 that says "Funny Quotes"?